tests/indent/java/constructor_with_arguments_on_multiple_lines.java

```diff
@@ -0,0 +1,5 @@
+public class Testo {
+  public Testo(
+    String a
+  ) {
+}
```

Comment:
tbh in this scenario here
The `}` is being matched with the closest `{` (the smallest range possible that it can be in)
so that's why it expects `  }` and not `}`

If you want to test dangling `{ }`, don't put the `}` in, add it through new line and check it

Reply:
Perfect, that did the trick. Thanks!